pkg/apis/resource/basic_view.go

```diff
@@ -33,12 +33,18 @@ import (
 	"github.com/seal-io/walrus/utils/validation"
 )
 
+type ResourceRunInfo struct {
+	Draft         bool              `json:"draft,default=false"`
```

Comment:
Draft controls the behavior of resource, shall we put it into resource run info?